src/System.Private.CoreLib/shared/System/Reflection/MethodBody.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Reflection
+{
+    public class MethodBody
+    {
+        internal MethodBase _methodBase;
```

Comment:
This property is causing the mismatch failures. I am not sure not how to handle this as this is being used by some other classes like runtimeMethodinfo and runtimeConstructorInfo… I tried using it through property or using override or new keyword in runtimeMethodbody, but they didn't work. @jkotas any ideas ?

Reply:
This field should be in RuntimeMethodBody.